<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="s3.ind.business.impl.AppointmentServiceImplTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2024-05-06T18:48:40" hostname="DESKTOP-88OFGPG" time="3.193">
  <properties/>
  <testcase name="createAppointment_WorkoutNotFound_ThrowsEntityNotFoundException()" classname="s3.ind.business.impl.AppointmentServiceImplTest" time="3.14"/>
  <testcase name="createAppointment_Successful()" classname="s3.ind.business.impl.AppointmentServiceImplTest" time="0.047"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
